lib/modules/header-widgets/index.js

```diff
@@ -129,6 +153,18 @@ module.exports = {
                     widget.imageId = imageId;
                     widget.formattedImageStyles = styleSchema.format(imageId, widget.imageStyles);
                 }
+                
+                if (widget.hasFixedIdeaCount) {
+                    widget.ideaCountParts = widget.fixedIdeaCount ? widget.fixedIdeaCount.substring(0, 3).split("") : [];
+                } else {
+                    widget.ideaCountParts = req.data.ideas ? req.data.ideas.length.toString().split("") : 0;
```

Comment:
The else part should also be an empty array

Reply:
Whoops, you're right!